Developers/Newcomers_Guide.rst

```diff
@@ -158,7 +158,7 @@ coala **clean** and **stable**.
 
```

Comment:
your body is pretty bad, try something better, such as:
"Fixes a previously broken link to an updated one."